docs/FAQ/FAQ-Wallet.md

```diff
@@ -1,133 +1,80 @@
+* [Do I have to use BTCPay Server wallet?](#do-i-have-to-use-btcpay-server-wallet)
+* [Missing payments in wallet?](#missing-payments-in-my-software-or-hardware-wallet)
 * [What is a derivation scheme?](#what-is-a-derivation-scheme)
+* [I don't see Lightning network payments in BTCPay wallet?](#i-dont-see-lightning-network-payments-in-btcpay-wallet)
 
-## What is BTCPay wallet?
+## What is BTCPay Server wallet?
 
-BTCPay has an internal wallet which you can use to see the transactions and even send the money out of it. The wallet works only with a supported hardware wallet, like Ledger Nano S.
+BTCPay Server has an internal wallet which you can use to preview incoming and outgoing transactions and manage your funds.
 
-You'll have to confirm and sign the transaction on your hardware wallet. BTCPay wallet is not a hot wallet. The private keys are secured inside your hardware wallet.
+It works like any other wallet, but has enhanced privacy features by default and also solves certain UX problems you may encounter when [using an existing wallet] with BTCPay Server. 
 
-## What is wallet re-scan in BTCPay?
+For more information on how to use the built-in wallet [check this page](/Wallet.md). To use the internal wallet, you first need to [set up the wallet](/WalletSetup.md) with your BTCPay store.
 
-The Rescan feature is best explained in a real-life situation:
+## How to set up my wallet with BTCPay Server?
 
-* You use a wallet outside of BTCPay (for example Ledger Nano S with the Ledger Live app)
-* You receive a payment worth 1BTC to your Nano S wallet.
-* After a while you decide to add the xpubkey you use with your Ledger to your BTCPay Server. You add this xpub as a derivation scheme in your Store.
-* You then find out that BTCPay has no information about transaction A, and shows 0 BTC balance while Ledger Live shows 1BTC.
+Check our in-depth documentation on [how to set up a wallet](/WalletSetup.md).
 
-This happens because when you use a new derivation scheme with your store, BTCPay Server doesn't know the past transactions of your wallet.
+## Can I use a hardware wallet with BTCPay Server?
 
-To solve this issue, you need to use the **Rescan feature**. The Rescan relies on Bitcoin Core 0.17.0's [scantxoutset](https://bitcoincore.org/en/doc/0.17.0/rpc/blockchain/scantxoutset/) to scan the current state of the blockchain (called UTXO Set) for coins belonging to your derivation scheme.
-Once the scan is complete, BTCPay Server will show the correct balance, along with some of past transactions of your wallet.
+The internal wallet has a [built in hardware wallet integration](/Vault.md). You can use a supported hardware wallet with the [BTCPay wallet](Wallet.md). 
 
-## Recommended external wallets?
+This  means that you're using a hardware wallet without leaking information to third-party apps or servers, since the wallet relays on the full node in your BTCPay.
```

Comment:
```suggestion
This  means that you're using a hardware wallet without leaking information to third-party apps or servers unlike the default software for the devices, since the wallet relies on the full node in your BTCPay.
```